aspnetcore/toc.yml

```diff
@@ -797,6 +797,10 @@ items:
             uid: mvc/controllers/filters
           - name: Razor SDK
             uid: razor-pages/sdk
+          - name: Razor breaking changes since .NET 8
+            href: ../_razor/docs/Compiler%20Breaking%20Changes%20-%20DotNet%208.md
+          - name: Razor breaking changes since .NET 9
+            href: ../_razor/docs/Compiler%20Breaking%20Changes%20-%20DotNet%209.md
           - name: View components
```

Comment:
Can you full paths
https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/toc.yml#L1461-L1463

Reply:
The absolute path works, but then the article isn't published in Learn, it's a link.

Here's how it displays for [roslyn breaking changes](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/breaking-changes/compiler%20breaking%20changes%20-%20dotnet%209)

Reply:
> The absolute path works, but then the article isn't published in Learn, it's a link.
> 

The markup for [Sample Docker images](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/toc.yml#L1461-L1463)

which you can see the two TOC entries below [VS tools](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/docker/visual-studio-tools-for-docker). The TOC entries are links that take you out of the aspnet/core doc set.

Can we live with TOC entries that are links?

> Here's how it displays for [roslyn breaking changes](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/breaking-changes/compiler%20breaking%20changes%20-%20dotnet%209)

That's an example from /dotnet, right. And that's how you want it to work, right?

Reply:
> That's an example from /dotnet, right. And that's how you want it to work, right?

Yes, exactly,